src/languages/java.md

```diff
@@ -1,3 +1,94 @@
 # Java
 
-Java support is in Alpha and not generally available. Contact sales [https://platform.sh/contact](https://platform.sh/contact) if you are interested in running Java based applications on Platform.sh.
+Java is the most popular language in today’s world billion of devices running on a Java platform; Java touched every field of development of software, java’s features make it the most powerful language in the current market of software development. The distributed application also quickly developed with the help of java. In current time lots of java’s framework presents which makes it more potent like Spring, Jakarta EE, Hibernate, JSP, Servlet, Spring Boot. Enterprise application development always preferred with java language only because of security and scalability features of the Java. For sure, what makes Java the most powerful platform in the globe is not the technology, but the vibrant and helpful community.
```

Comment:
To me this still seems long for what it's trying to accomplish. Maybe something as short as:

Java is a general-purpose programming language, and one of the most popular in the world today. Platform.sh supports Java runtimes that can be used with build management tools such as Gradle, Maven, and Ant. 

I defer to @Crell.

Reply:
Thank you, I'll paste this.